tableauserverclient/server/endpoint/auth_endpoint.py

```diff
@@ -7,6 +7,9 @@
 
 
 class Auth(Endpoint):
+    def __init__(self):
+        super(Auth, self).__init__(None)
```

Comment:
Keyword argument? I don't know what `None` means it kinda makes me think it's passing some fake value